src/pc/sdl2.c

```diff
@@ -10,6 +10,7 @@
 #endif
 #include "pc.h"
 #include "sdl_defs.h"
+#include "macros.h"
```

Comment:
Could you rename `macros.c` and `macros.h` into `sdl2_macros.c` and `sdl2_macros.h`? so we remember to keep them isolated